libproject/test/libprojecttest.cpp

```diff
@@ -298,6 +280,9 @@ class TestRemoveSubprojects : public TestSkeleton {
     case_6_pathToMainFileWhichWillNotBeLoaded,
```

Comment:
*#bug*
case_6_
case_7_
e.t.c
Are bad names - them don't give info about what is it.